tests/integration/commands/call_command_test.py

```diff
@@ -36,6 +38,42 @@ async def test_call(
     assert response.res == 6  # type: ignore
 
 
+async def test_call_inputs(
+    protostar: ProtostarFixture,
+    devnet_gateway_url: str,
+):
+    deploy_response = await protostar.deploy(
+        contract=protostar.project_root_path / "build" / "main.json",
+        gateway_url=devnet_gateway_url,
+    )
+
+    response = await protostar.call(
```

Comment:
Could you please move this logic to separate test case?

Reply:
[address feedback in tests](https://github.com/software-mansion/protostar/pull/1140/commits/dd3248b2c6e8d02b1338c84e78c81b39e1ca0225)